main/webapp/modules/core/langs/translation-en.json

```
@@ -443,6 +443,7 @@
     "core-recon/cell-contains": "Each cell contains:",
     "core-recon/max-candidates": "Maximum number of candidates to return",
     "core-recon/service-title": "Services",
+    "core-recon/url-already-registered":"This reconciliation service already exists.",
```

Apologies for the conflicting reviews, but I think the Spanish version has better semantics. I'd suggest

```suggestion
    "core-recon/url-already-registered":"This reconciliation service is already registered.",
```